Thanks @OpenCode 

please check

```
stock_picking_package_preparation_line/models/ir_config.py:5:1: F401 'api' imported but unused
```

and test error

```
AccessError: ('AccessError', 'Sorry, you are not allowed to access this document. Please contact your system administrator if you think this is an error.\n\n(Document model: stock.config.settings)')
```

--- a/stock_picking_package_preparation_line/models/ir_config.py
+++ b/stock_picking_package_preparation_line/models/ir_config.py
@@ -2,7 +2,7 @@
 # © 2015 - Francesco Apruzzese <f.apruzzese@apuliasoftware.it>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from openerp import models, fields, api
+from openerp import models, fields
 
 
 class ResCompany(models.Model):
@@ -14,7 +14,7 @@ class ResCompany(models.Model):
         string='Default Picking Type used in package preparation')
 
 
-class StockPickingPackagePreparationLineSettings(models.Model):
+class StockPickingPackagePreparationLineSettings(models.TransientModel):
 
     _inherit = 'stock.config.settings'
 

